tests/ui/fn/suggest-return-closure.rs

```diff
@@ -19,6 +19,7 @@ fn fn_mut() -> _ {
     let x = String::new();
     //~^ HELP: consider changing this to be mutable
     //~| NOTE binding `x` declared here
+    //~| SUGGESTION mut
```

Comment:
Rule of thumb to maximize annotations while minimizing test changes: if just one additional `SUGGESTION` is required, then I add it, otherwise I add `//@ dont-require-annotations:SUGGESTION`.
I'm going to do the same thing when adding `HELP`s and `NOTE`s in a next PR.